CoreUtilFramework/Network/APIResponse.swift

```diff
@@ -43,6 +47,11 @@ public struct ApiWarning {
     }
     
     public init() {}
+    
+    public func messageWithErrorCode(messageTemplate:String = "%@ (%@)")->String?{
+        guard let message = message, let code = code else { return nil }
+        return String(format: messageTemplate, message, code)
+    }
```

Comment:
code her zaman bir hataya işaret etmediği için metod ismi refacto edilebilir.

```suggestion
    public func messageWithCode(messageTemplate: String = "%@ (%@)") -> String? {
        guard let message = message, let code = code else { return nil }
        return String(format: messageTemplate, message, code)
    }
```